tc/sgx/common/packages/parson/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Intel Corporation
```

Comment:
2020?

Reply:
I will take care of updating copyright for these files in my new PR.